official/projects/vit/modeling/vit.py

```diff
@@ -172,6 +172,22 @@ def call(self, inputs, training=None):
       x = encoder_layer(x, training=training)
     x = self._norm(x)
     return x
+  def get_config(self):
+    config = {
+        'num_layers': self._num_layers,
+        'mlp_dim': self._mlp_dim,
+        'num_heads': self._num_heads,
+        'dropout_rate': self._dropout_rate,
+        'attention_dropout_rate': self._attention_dropout_rate,
+        'kernel_regularizer': self._kernel_regularizer,
+        'inputs_positions': self._inputs_positions,
+        'init_stochastic_depth_rate': self._init_stochastic_depth_rate,
+        'kernel_initializer': self._kernel_initializer,
+        'add_pos_embed': self._add_pos_embed,
+    }
+    base_config = super().get_config()
+    return dict(list(base_config.items()) + list(config.items()))
+
```

Comment:
Nit: can you remove one empty line at 190 to 192, and add one empty line above "def get_config"? Thanks!